plugins/doc_fragments/auth.py

```diff
@@ -52,8 +52,9 @@ class ModuleDocFragment(object):
         description:
           - For token auth, will perform a C(lookup-self) operation to determine the token's validity before using it.
           - Disable if your token does not have the C(lookup-self) capability.
+          - The default value is C(True).
```

Comment:
In documentation Ansible usually uses the lower-case versions.
```suggestion
          - The default value is C(true).
```